src/panels/config/cloud/dialog-cloud-already-connected/dialog-cloud-already-connected.ts

```diff
@@ -0,0 +1,170 @@
+  }
+
+  public closeDialog() {
+    this._params?.closeDialog();
+    this._params = undefined;
+    this._obfuscateIp = true;
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  protected render() {
+    if (!this._params) {
+      return nothing;
+    }
+    const { details } = this._params;
+
+    return html`
+      <ha-dialog
+        open
+        @closed=${this.closeDialog}
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize(
+            "ui.panel.config.cloud.dialog_already_connected.heading"
+          )
+        )}
+      >
+        <div>
+          <p>
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.description"
+            )}
+          </p>
+          <b>
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.other_home_assistant"
+            )}
+          </b>
+          <div class="instance-details">
+            <div class="instance-detail">
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.ip_address"
+                )}:
+              </p>
+              <div class="obfuscated">
+                <p>
+                  ${this._obfuscateIp
+                    ? obfuscateUrl(details.remote_ip_address)
+                    : details.remote_ip_address}
+                </p>
+
+                <ha-icon-button
+                  class="toggle-unmasked-url"
+                  toggles
+                  .label=${this.hass.localize(
+                    `ui.panel.config.cloud.dialog_already_connected.obfuscated_ip.${this._obfuscateIp ? "hide" : "show"}`
+                  )}
+                  @click=${this._toggleObfuscateIp}
+                  .path=${this._obfuscateIp ? mdiEye : mdiEyeOff}
+                ></ha-icon-button>
+              </div>
+            </div>
+            <div class="instance-detail">
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.connected_at"
+                )}:
+              </p>
+              <p>
+                ${formatDateTime(
+                  new Date(details.connected_at),
+                  this.hass.locale,
+                  this.hass.config
+                )}
+              </p>
+            </div>
+          </div>
+          <ha-alert
+            alert-type="info"
+            .title=${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.info_backups.title"
+            )}
+          >
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.info_backups.description"
+            )}
+          </ha-alert>
+        </div>
+
+        <ha-button @click=${this.closeDialog} slot="secondaryAction">
+          ${this.hass!.localize(
+            "ui.panel.config.cloud.dialog_already_connected.close"
```

Comment:
Close is correct but what about say `Cancel` here? As I cancel the login process when I do this.

Moreover we should use `ui.common.close`

Reply:
Are the rules for when to use common and not written somewhere?
the dialog I based this on, does the same..

Reply:
If it's in ui.common you should use it because this is always loaded, except you are in supervisor or onboarding panel because they just load their own translations